app/views/shopping_shared/_tabs.html.haml

```diff
@@ -1,11 +1,13 @@
-- shop_tabs.each do |tab|
-  = render "shopping_shared/tabs/#{tab[:name]}"
+- if (@order.andand.distributor || current_distributor) == current_distributor
```

Comment:
why is this needed? if I just switched shops I still want to see the tabs for the new shop, right?

Reply:
If you switched shops then `current_distributor` should be the new shop.

This is a quick fix for another bug. I can't find the issue right now but if your current distributor is ShopA and then you look at an order receipt for ShopB then the tabs use current distributor and looking up contact details shows you ShopA. So this `if` clause is not fixing the problem but avoiding it by hiding the tabs in that case. Better than before, I think.

Reply:
This bit of logic was present in the original `details.html` partial, I just moved it here when splitting the file into `header.html` and `tabs.html`.

https://github.com/openfoodfoundation/openfoodnetwork/pull/4707/files#diff-d2ee7c7d99b77ac7210e1d930482692eR1
https://github.com/openfoodfoundation/openfoodnetwork/pull/4707/files#diff-d2ee7c7d99b77ac7210e1d930482692eL18

Reply:
ah, makes sense! Thanks for explaning Maikel. I think this "looking at an order receipt for ShopB" deserves a comment in the code.